### DISCUSSION TITLE
copilot authorize problem

### DISCUSSION THREAD

**Discussion by zanechou** [community/community#14590] [Copilot Conversations]
when I use copilot ，It shows me 
![1649464518(1)](https://user-images.githubusercontent.com/29723352/162549503-516d43e9-62d4-4a7e-819a-1b74f67c0714.jpg)
and I click “确定”，and It shows me：
![1649463983(1)](https://user-images.githubusercontent.com/29723352/162549556-7f935777-f6be-4cdc-a69d-ade48215eb50.jpg)
![1649463998(1)](https://user-images.githubusercontent.com/29723352/162549564-5ed1f3e0-3b9e-46b8-8b43-a159c1ebf3c2.jpg)
After do this，It still me the firist picture。

**Comment by virginity-is-cool** [comment]
same issue, fuck you american

**Comment by essn** [comment]
If anyone is running into this problem on OSX I managed to get around it by deleting all Github-related items in Keychain Access and rebooting my machine.

After doing some spring cleaning of OAuth tokens I think it was holding onto references to old tokens that no longer existed.
